pyDeltaRCM/iteration_tools.py

```diff
@@ -78,17 +87,36 @@ def finalize_timestep(self):
 
         self.H_SL = self._H_SL + self._SLR * self._dt
 
+    def log_info(self, message, verbosity=0):
+        """Log message dependent on verbosity settings.
+
+        Parameters
+        ----------
+        message : :obj:`str`
+            Message string to write to the log as info.
+
+        verbosity : :obj:`int`, optional
+            Verbosity threshold, whether to write the message to the log or
+            not. Default value is `0`, or i.e. to always log.
+        """
+        if self._verbose >= verbosity:
+            self.logger.info(message)
```

Comment:
I noticed that this re-factoring removed the printing out of log messages when `verbosity==2`. I assume this was by design, but bring it up here because if we wanted to enable (optional) printing of log messages out, we could probably stick an if-statement here that does `print(message)` if verbosity is at a certain level (similar to what is done for `log_model_time`).

Reply:
thanks for taking a look! Yeah, it was by design. I decided that the printing should probably be kept to a minimum; after all, that is the point of having a log file.

It would be easy enough to support printing with some option in the yaml, now that all calls to the log are in this one method.

in yaml:
```yaml
stdout: True
```

then in `log_info()`:
```python
if self._verbose >= verbosity:
    self.logger.info(message) 
    if self._stdout:
        print(message)
```

Of course, this means something similar should probably be done to `log_info` for the various warnings and exceptions write to the log/raise directly...

Idk I don't feel strongly about getting this done now, but it would be good to jot some ideas down in an issue. What do you think?

Reply:
> I decided that the printing should probably be kept to a minimum; after all, that is the point of having a log file.

Yes I agree with this sentiment, I don't think we need to open up an issue to add that type of functionality - I don't necessarily see what problem printing would solve as a log file will exist so it's hard to justify adding more work, code, and yaml parameters.

Reply:
right on. Works for me.